kv/replica_slice_test.go

```diff
@@ -110,3 +111,22 @@ func TestReplicaSetMoveToFront(t *testing.T) {
 		t.Errorf("expected order %s, got %s", exp, stores)
 	}
 }
+
+func TestReplicaSetRandomOrder(t *testing.T) {
```

Comment:
It doesn't look like this test even calls `SortRandomOrder`. `MoveToFront` just moves one element; it doesn't shuffle the rest of the array.